juju/juju
# Feature/add device2bundlechanges

## Description of change

*Please replace with a description about why this change is needed?*

added `Device` support for `bundle` `facade`.

## QA steps

*Please replace with how we can verify that the change works?*

Client only, needs back-end support

## Documentation changes

It only includes client side changes for device support on bundle deploy.
Document changes will be required once back-end done.
